docs/4.0/docs/examples/use-private-registry.md

```diff
@@ -0,0 +1,69 @@
+# how to use sealos private registry
+
+## where it is
+
+the sealos private registry running in the first nodes of the cluster,where you create your cluster, you can use the follow comand check it:
+
+```
+root@node01:~# nerdctl ps
+CONTAINER ID    IMAGE                               COMMAND                   CREATED         STATUS    PORTS    NAMES
+eb772a8cc788    docker.io/library/registry:2.7.1    "/entrypoint.sh /etc…"    22 hours ago    Up                 sealos-registry 
+```
+
+the registry images data store in  follow directory:
+
+```
+/var/lib/sealos/data/default/rootfs/registry/
+```
+
+## login to registry
+
+sealos private registry run with http and `--net host ` option，you should config insecure-registry at local and connect with first node ipaddress，example of docker client:

```

Comment:
```suggestion
sealos private registry runs with HTTP and `--net host` option. You should configure `insecure-registries` at local and connect with the first node's IP address. Example of docker client:
```

Please make sure all the punctuations are English punctuations, not the Chinese ones.